recipe/bld.bat

```diff
@@ -1,2 +1,9 @@
+:: Link against libcurl package instead of statically bundling it.
+sed -i src/Makevars.win ^
+    -e "/TARGET/d" ^
+    -e "/winlibs\.R/d" ^
+    -e "s|-L\S*RWINLIB\S*|-L%LIBRARY_LIB:\=/%|g" ^
+    -e "s|-I\S*RWINLIB\S*|-I%LIBRARY_INC:\=/%|g" ^
+    -e "s|-DCURL_STATICLIB\s*||g"
```

Comment:
The builds also ran through without this.
We prefer to dynamically link to our packages instead of bundling static libraries, but also feel free to amend/remove this if it causes trouble.